rqt_py_common/src/rqt_py_common/message_helpers.py

```diff
@@ -41,19 +42,126 @@
 SRV_MODE = 'srv'
 ACTION_MODE = 'action'
 
+ROSIDL_FILTERS = {
+    MSG_MODE: lambda n: n.endswith('.msg'),
+    SRV_MODE: lambda n: n.endswith('.srv'),
+    ACTION_MODE: lambda n: n.startswith('action/') and n.endswith('.idl')
```

Comment:
The current code base doesn't have `.idl` file. Therefore this should check for the extension `.action`.

Reply:
Oh strange. I guess I'm working from some older branches. I was hoping for an .action interface to be added. 

Updated